### PR TITLE
Update adding_a_cask.md

### PR DESCRIPTION
**Pull request by miccal**

### CODE REVIEW

File: doc/development/adding_a_cask.md [hunk 1]
```diff
@@ -205,7 +205,7 @@ Please submit these to [caskroom/homebrew-unofficial](http://github.com/caskroom
 
 ### Fonts
 
-Font casks live in the [caskroom/homebrew-fonts](https://github.com/caskroom/homebrew-fonts) repository. See the font repo [CONTRIBUTING.md](../../CONTRIBUTING.md)
```
Comment (vitorgalvao) [inline doc/development/adding_a_cask.md]: When refering to other documentation, I’d rather keep the paths as relative.

File: doc/development/adding_a_cask.md [hunk 2]
```diff
@@ -166,7 +166,7 @@ We maintain separate Taps for different types of binaries. Our nomenclature is:
 + **Beta, Development, Unstable**: Subsequent versions to **stable**, yet incomplete and under development, aiming to eventually become the new **stable**.
 + **Nightly**: Constantly up-to-date versions of the current development state.
 + **Legacy**: Any **stable** version that is not the most recent.
-+ **Alternative**: Alternative edition of an existing app, by the same vendor (developer editions, community editions, pro editions, …).
```
Comment (vitorgalvao) [inline doc/development/adding_a_cask.md]: I’d rather keep the elipsis.

**Comment by miccal** [comment]
Updated.

File: doc/development/adding_a_cask.md [hunk 3]
```diff
@@ -166,7 +166,7 @@ We maintain separate Taps for different types of binaries. Our nomenclature is:
...
++ **Alternative**: Alternative edition of an existing app, by the same vendor (developer editions, community editions, pro editions, ...).
```
Comment (vitorgalvao) [inline doc/development/adding_a_cask.md]: Wrong character. Should be `…` (1 character), and not `...` (3 characters). I can fix that, though.

File: doc/development/adding_a_cask.md [hunk 4]
```diff
@@ -245,7 +245,7 @@ See [a note about git commit messages](http://tbaggery.com/2008/04/19/a-note-abo
 
 The first line of a commit message becomes the **title** of a pull request on GitHub, like the subject line of an email. Including the key info in the first line will help us respond faster to your pull.
 
-For Cask commits in the Homebrew-Cask project, we like to include the Application name, version number (or `:latest`), and purpose of the commit in the first line.
+For Cask commits in the Homebrew-Cask project, we like to include the application name, version number (or `:latest`), and purpose of the commit in the first line.
```
Comment (vitorgalvao) [inline doc/development/adding_a_cask.md]: You usually uppercase the word, so why downcase it here?

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in b964a9724e6b9c09147e8e6500ec27e0622fe95d. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**